src/main/config/config_eeprom.c

```diff
@@ -284,10 +284,7 @@ bool loadEEPROMFromSDCard(void)
 #endif
 
 #ifdef CONFIG_IN_FILE
-void loadEEPROMFromFile(void)
-{
-    FLASH_Unlock(); // load existing config file into eepromData
-}
+extern void loadEEPROMFromFile(void);
```

Comment:
Never declare external function in .c file, use .h included both in file defining it and file using it. Otherwise compiler won't check function signature, possibly leading to nasty, hard to find bugs.

Reply:
done. moved to *_impl.h